community/dnsdist/APKBUILD

```diff
@@ -1,8 +1,8 @@
 # Contributor: Vince Mele <vmele@inoc.com>
-# Maintainer: Vince Mele <vmele@inoc.com>
```

Comment:
Did you contact Vince about this?

Reply:
Not since the first PR they submitted. See https://github.com/alpinelinux/aports/pull/3321